config/notarize.js

```diff
@@ -0,0 +1,19 @@
+const { notarize } = require('electron-notarize');
+
+exports.default = async function notarizing(context) {
+  const { electronPlatformName, appOutDir } = context;
+  if (electronPlatformName !== 'darwin') {
+    return;
+  }
+
+  const appName = context.packager.appInfo.productFilename;
+  console.log('notarizing app');
+
+  await notarize({
+    appBundleId: 'com.electron.codeorg-maker-app',
+    appPath: `${appOutDir}/${appName}.app`,
+    appleId: process.env.APPLEID,
+    appleIdPassword: process.env.APPLEIDPASS,
+  });
+  return;
```

Comment:
it looks like eslint doesn't like this `return` statement:

<img width="666" alt="Screen Shot 2021-05-27 at 8 49 22 AM" src="https://user-images.githubusercontent.com/9812299/119857489-758f6300-bec8-11eb-8480-f2130560c953.png">

Reply:
Indeed. Earlier it failed on [`no-return-await`](https://eslint.org/docs/rules/no-return-await) but my fix was short-sighted; in this case I should still return the result from the `notarize()` call.